.github/workflows/ci.yml

```diff
@@ -282,8 +282,8 @@ jobs:
       env:
         DOCKER_CLI_EXPERIMENTAL: enabled
       run: |
-        wget https://raw.githubusercontent.com/mattermost/mattermost-server/master/build/Dockerfile
-        wget https://raw.githubusercontent.com/mattermost/mattermost-server/master/build/entrypoint.sh
+        wget https://raw.githubusercontent.com/mattermost/mattermost-server/master/server/build/Dockerfile
+        wget https://raw.githubusercontent.com/mattermost/mattermost-server/master/server/build/entrypoint.sh
```

Comment:
@kostaspt, should we merge this change in the cloud branch?

Reply:
Yes, I think so because those docker-related files will be there from now on. The `cloud` branch for webapp, as long we're using that for a release will need this change. Soon we'll not be using this repo for the `cloud` release.